.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
```

Comment:
maybe specify targets to make sure ie8 is included?
presets: [['@babel/preset-env', {targets: {browsers: ['ie >= 8']}}]],